sdk/metricsadvisor/ai-metrics-advisor/review/ai-metrics-advisor.api.md

```diff
@@ -340,7 +342,7 @@ export interface DataPointAnomaly {
 }
 
 // @public
-export type DataSourceType = "AzureApplicationInsights" | "AzureBlob" | "AzureCosmosDB" | "AzureDataExplorer" | "AzureDataLakeStorageGen2" | "AzureTable" | "Elasticsearch" | "HttpRequest" | "InfluxDB" | "MongoDB" | "MySql" | "PostgreSql" | "SqlServer";
+export type DataSourceType = string;
```

Comment:
It feels that we need more to be able to use this as extensible enums? The swagger defines a list of possible data source using polymophic discriminator. Code gen creates a list of data source types that doesn't include `string` so we couldn't really extend this to be compatible

https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/metricsadvisor/ai-metrics-advisor/src/generated/models/index.ts#L684-L697

@joheredi any ideas?